commands/module-list.md

```diff
@@ -0,0 +1,10 @@
+Returns information about the modules loaded to the server.
+
+@return
+
+@array-reply: list of loaded modules. Each element in the list represents a
```

Comment:
Not a big deal, but I would probably avoid the term 'list' as it can be confused for the data structure.

Reply:
As it already in the documentation in other commands at the same place (see `HGETALL`). I'd rather stick with the "standard" than be "technically correct" (debest kind of!)